change/@ni-nimble-components-0aeacb05-bfbf-47eb-8e8d-d47e996f3004.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
@fredvisser is this PR ready to merge or are you waiting for a response to this question? https://github.com/ni/nimble/pull/2040#issuecomment-2079904402

Reply:
Let's merge it - 🎉